.coderabbit.yaml

```
@@ -0,0 +1,73 @@
+  
+  # Path filters and instructions
+  path_filters: []  # Add glob patterns to include/exclude files
+  path_instructions: []  # Add path-specific review instructions
+  
+  # Auto review settings
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    drafts: false
+    base_branches:
+     - help-center-task-2
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)